jobs/scripts/5.x-at.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-pip install -r requirements.txt
-buildout buildout:git-clone-depth=1 -c core.cfg
-bin/alltests-at --xml
+pip install -Ur requirements.txt
```

Comment:
The ``-U`` here is superfluous as ``requirements.txt`` pins everything needed and has no more dependencies, and on top of all, we got bitten too many times with an unexpected upgrade of setuptools, so no, I'm no fan of newest/update blindly :smile: